constitution.md

```diff
@@ -0,0 +1,684 @@
+
+### _PrePur_: Purpose of the organisation
+
+_PrePur-1_: Linux Australia is established with the purpose of advancing the open source and related technologies in Australia:
+
+* conducting and publishing research into improvements to the processes used in open source and related technologies,
+* working with government at all levels to ensure that the interests of the open source and related technologies are represented in regard to the public decision-making process, and
+* providing a forum for all people engaged in the open source and related technologies to discuss best practice and enhancing the future of the software and computing.
+
+### _PreObj_: Objects of the organisation
+
+_PreObj-1_: The objects for which the association is established are:
+
+* to facilitate the communication of information relating to inventions, results, techniques, and governance of open source and related open technologies;
+* to support open source and related open technologies projects and the contributors to open source and related open technologies;
+* to support and promote the adoption of open source and related open technologies;
+* to support and promote awareness and understanding of open source and related open technologies;
+* to conduct open source and related open technologies advocacy with the government, business and civil society;
+* to create a network for related local and online open source and related open technologies communities.
+
+# Part 2 - Members of association
+
+### _MemGen_: Membership generally
+
+_MemGen-1_: An individual is taken to be a member of the association if:
+
+* the person applied to be a member under clause _MemApp-1_ and the application has been approved, or
+* the person was 1 of the individuals on whose behalf an application for registration of the association was made under the Act, section 6(1)(a), or
+* the person was:
+  - for an unincorporated body registered as the association - a member of the body immediately before the registration, or
+  - for a registrable corporation registered as the association - a member of the corporation immediately before the registration, or
+  - for an association that was amalgamated to form the relevant association - a member of the existing association immediately before the amalgamation.
+
+_MemGen-2_: A person who is not an individual is not eligible to be a member of the association.
+
+### _MemApp_: Membership applications
+
+_MemApp-1_: An application by a person to be a member of the association must be:
+
+- made in writing, and
+- in the form determined by the committee, and
+- lodged with the secretary.
+
+_MemApp-2_: The committee may determine that an application may be made or lodged by email or other electronic means.
+
+_MemApp-3_: The secretary must determine whether to approve or reject the application or refer the nomination to the committee which is to determine whether to approve or to reject the nomination.
```

Comment:
❓ Is there a need to provide the committee with the ability to review the secretary's decision?

Reply:
I think the final decision has to lie with the committee, not one person.

However, in practice with LA there one bunny doing the work (who is not always the secretary), they delegate contentious ones (like overseas) to the committee and approve the rest on the spot.  I tried to allow for that here.

It would be a rare organisation that had the entire committee review every application.  I suspect unheard of in one with as many members as LA.  The existing wording in the model constitution must be good enough to cover that.  I think I'll just revert to the original text.